apps/app/src/views/routes/Settings/components/users/UserItemActions.tsx

```diff
@@ -63,6 +83,54 @@ export const UserItemActions: React.FC<UserItemActionsProps> = ({ user }) => {
           <MenuItem icon={<FiEdit fontSize="1.2em" />} onClick={onOpen}>
             Edit Roles
           </MenuItem>
+          <MenuItem
+            icon={<FiTrash fontSize="1.3em" color="red" />}
+            disabled={deleteLoading}
+            onClick={async () => {
+              const decision = await confirmWrapper('Remove user?', {
+                description: <Text mb={2}>This will remove user from organization.</Text>,
```

Comment:
Also, text should say `This will remove the user from the organization`, or better: `{{ user.name }} will be removed from {{ organization.name }}`.

Reply:
It feels like incosistency that 2 deletes in 2 screens behave differently. sure I will change

Reply:
done